src/main/java/com/zaxxer/hikari/HikariJNDIFactory.java

```diff
@@ -40,31 +40,26 @@
 public class HikariJNDIFactory implements ObjectFactory
 {
    @Override
-   synchronized public Object getObjectInstance(Object obj, Name name, Context nameCtx, Hashtable<?, ?> environment) throws Exception
-   {
+   synchronized public Object getObjectInstance(Object obj, Name name, Context nameCtx, Hashtable<?, ?> environment) throws Exception {
```

Comment:
HikariCP's code style is open brace on a new line after method declaration, and open brace on the same line inside in method.

Reply:
DONE